alembic/versions/495e6c766315_reload_sunset_date_function.py

```diff
@@ -0,0 +1,22 @@
+"""reload sunset_date function
+
+Revision ID: 495e6c766315
+Revises: 5bafdc19756c
+Create Date: 2016-10-31 12:46:22.476356
+
+"""
+
+from alembic import op
+from socorrolib.lib.migrations import load_stored_proc
+
+# revision identifiers, used by Alembic.
+revision = '495e6c766315'
+down_revision = '5bafdc19756c'
+
+
+def upgrade():
+    load_stored_proc(op, ['sunset_date.sql'])
+
+
+def downgrade():
+    load_stored_proc(op, ['sunset_date.sql'])
```

Comment:
https://bitbucket.org/zzzeek/alembic/issues/119/thought-experiments-in-managing-stored

Went looking and found selena wondering about this from back when we first adopted alembic. The answer to my question is "no". It's basically on us to create a different `sunset_data.sql` file that we can load in the downgrade or use the [replaceable object pattern](http://alembic.zzzcomputing.com/en/latest/cookbook.html#replaceable-objects).